arclightning_backend/src/router.rs

```diff
@@ -375,7 +383,7 @@ impl Router {
         );
 
         if requested_path == root {
-            *request.uri_mut() = hyper::Uri::from_static("/index.html");
+            *request.uri_mut() = hyper::Uri::from_static("/start.html");
```

Comment:
i didn't even have to ask, you just did this. gg

Reply:
gg